go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/ceph/ceph-csi
 
-go 1.23.0
+go 1.23.1
+
+toolchain go1.23.4
```

Comment:
Somebody please leave a comment about the `toolchain` parameter in this file. I think there was a problem with adding it, but I don't remember what it was... @ceph/ceph-csi-contributors

Reply:
AFAIK there was no problem with toolchain as it installs the golang based on the toolchain version and it's not a hard dependency but it's a soft dependency to build cephcsi.